src/components/calculator/PointsDisplay.tsx

```diff
@@ -107,21 +110,34 @@ function howRisky(points: number, budget: number): string[] {
   }
 }
 
-const budgetConsumption = (points: number, budget: number) => {
+const budgetConsumption = (
+  points: number,
+  budget: number,
+  repeatedEvent: boolean,
+) => {
+  if (repeatedEvent) {
```

Comment:
This function has some redundant markup. Perhaps we can refactor it to a single `return` at the end with `<p>{textVariableSetByAConditional}</p>`